docs/appendix.rst

```diff
@@ -524,76 +509,190 @@ To fix this, install the Python "wheel" package::
+   urllib3 exists it will not be upgraded to 2.0+ in the reinstallation of
+   pywbem.
 
 2. If the current version of urllib3 is greater than 2.0, the previous version
    of urllib3 (ex. version 1.26.5) can be installed (ex. ``pip install
-   urllib3<2.0``)
+   urllib3<2.0``).
 
-3. Adding TLS 1.2 support to the server side (preferred) or by lowering the
-    minimum TLS level OpenSSL requires on the client
-    side (which lowers security). The latter can be done by changing the
-    ``MinProtocol`` parameter in the OpenSSL config file on the client OS
-    (typically ``/etc/ssl/openssl.cnf`` on Linux and OS-X,
-    and ``C:\OpenSSL-Win64\openssl.cnf`` on Windows).
-    At the end of the file there is::
+3. Adding TLS 1.2 support to the WBEM server side (preferred) or lowering the
+   minimum TLS level required on the client side (which lowers security). With
+   OpenSSL the latter can be done by changing the ``MinProtocol`` parameter in
+   the OpenSSL config file on the client OS (typically ``/etc/ssl/openssl.cnf``
+   on Linux and OS-X, and ``C:\OpenSSL-Win64\openssl.cnf`` on Windows).
+
+   At the end of the file set the following::
 
         [system_default_sect]
         MinProtocol = TLSv1.2
         CipherString = DEFAULT@SECLEVEL=2
 
+4. If the issue is the use of LibreSSL (ex. MacOS) as the OS level SSL
+   implementation, the issue may be that urllib3 before version 2.0.3 did not
+   support LibreSSL.  See  `urllib3 issue 2020: <https://github.com/urllib3/urllib3/issues/3020>`_
+   for discussion of this issue.
+
+
+.. _ImportError urllib3 v2.0 only supports OpenSSL 1.1.1+:
+
+ImportError urllib3 v2.0 only supports OpenSSL 1.1.1+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: single LibreSSL
+.. index:: pair; troubleshooting: OpenSSL
+.. index:: single LibreSSL
+.. index:: LibreSSL macOS
+.. index:: single: TLS
+.. index:: pair; ImportError urllib3: OpenSSL
+
+The 'urllib3' Python package version 2.0 removed support for LibreSSL and
+some other SSL implementations and requires OpenSSL 1.1.1 or higher.  LibreSSL
+support was restored to 'urllib3' in version 2.0.3.
+
+The 'pywbem' package uses the 'requests' package which uses 'urllib3';
+neither 'pywbem' nor 'requests' pins 'urllib3' to stay below version 2.0.
+
+Therefore, upgrading Python packages, with a Python that does not support
+OpenSSL 1.1.1 or higher or other SSL implementations, causes the following
+exception raised by urllib3:
+
+    ImportError: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the ‘ssl’ module is compiled with LibreSSL 2.8.3.
+    See: https://github.com/urllib3/urllib3/issues/2168
+
+This can happen for example on macOS with the system Python of
+macOS as the basis for a Python virtual environment and installing pywbem
+into that virtual environment, which typically installs the latest available
+versions of dependent packages, and thus may install urllib3 with a version 2.0
+or later.
```

Comment:
This paragraph is mostly a duplication of the paragraph "Therefore, upgrading Python packages,...".